.coveragerc

```diff
@@ -451,6 +451,7 @@ omit =
     homeassistant/components/sensor/fritzbox_netmonitor.py
     homeassistant/components/sensor/gearbest.py
     homeassistant/components/sensor/geizhals.py
+    homeassistant/components/sensor/github.py
```

Comment:
Remove unrelated changes, including this the xs1 and zha changes below.

Reply:
I'm not sure how to undo this. I don't need to have the file changed, so I copied the coveragerc file from the dev branch of my fork is this okay or is there a better way?